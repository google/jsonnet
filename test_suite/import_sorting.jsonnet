local c = import "c.jsonnet"; // xccc
local b = import "b.jsonnet"; // xbbb
local a = import "a.jsonnet"; // xaaa
// end of imports
true
