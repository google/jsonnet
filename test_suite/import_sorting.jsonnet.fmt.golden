local a = import "a.jsonnet";  // xaaa
local b = import "b.jsonnet";  // xbbb
local c = import "c.jsonnet";  // xccc
// end of imports
true
